src/DIRAC/ConfigurationSystem/Client/PathFinder.py

```diff
@@ -13,33 +13,57 @@
 
 
 def getDIRACSetup():
+  """ Get DIRAC default setup name
+
+      :return: str
+  """
   return gConfigurationData.extractOptionFromCFG("/DIRAC/Setup")
 
 
-def divideFullName(entityName):
-  fields = [field.strip() for field in entityName.split("/")]
-  if len(fields) < 2:
-    raise RuntimeError("Service (%s) name must be with the form system/service" % entityName)
-  return tuple(fields)
+def divideFullName(entityName, second=None):
+  """ Convert component full name to tuple
+
+      :param str entityName: component full name, e.g.: 'Framework/ProxyManager'
+      :param str second: component name
+
+      :return: tuple -- contain system and component name
+  """
+  if entityName and '/' not in entityName and second:
+    return (entityName, second)
+  fields = [field.strip() for field in entityName.split("/") if field.strip()]
+  if len(fields) == 2:
+    return tuple(fields)
+  raise RuntimeError("Service (%s) name must be with the form system/service" % entityName)
+
+
+def getSystemInstance(system, setup=False):
+  """ Find system instance name
 
+      :param str system: system name
+      :param str setup: setup name
 
-def getSystemInstance(systemName, setup=False):
-  if not setup:
-    setup = gConfigurationData.extractOptionFromCFG("/DIRAC/Setup")
-  optionPath = "/DIRAC/Setups/%s/%s" % (setup, systemName)
+      :return: str
+  """
+  optionPath = "/DIRAC/Setups/%s/%s" % (setup or getDIRACSetup(), system)
   instance = gConfigurationData.extractOptionFromCFG(optionPath)
-  if instance:
-    return instance
-  else:
+  if not instance:
     raise RuntimeError("Option %s is not defined" % optionPath)
+  return instance
 
 
-def getSystemSection(serviceName, serviceTuple=False, instance=False, setup=False):
-  if not serviceTuple:
-    serviceTuple = divideFullName(serviceName)
-  if not instance:
-    instance = getSystemInstance(serviceTuple[0], setup=setup)
-  return "/Systems/%s/%s" % (serviceTuple[0], instance)
+# TODO: serviceTuple here for backward compatibility and must be deleted in the next release(v7r4)
```

Comment:
Please make an issue, or create directly the PR yourself now.

Reply:
I found only one case on all code where the tuple was transferred therefore I think it can be removed simply considering that changes go in integration